Directory.Build.props

```diff
@@ -2,6 +2,7 @@
 	<PropertyGroup>
 		<!-- Disables the transitive restore of packages like Microsoft.AspNetCore.App.Ref, Microsoft.WindowsDesktop.App.Ref -->
 		<DisableTransitiveFrameworkReferenceDownloads>true</DisableTransitiveFrameworkReferenceDownloads>
+		<MonoCecilPackageVersion>0.11.5</MonoCecilPackageVersion>
```

Comment:
Using centralized package versions might be a better approach: https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management